src/plopp/tools.py

```diff
@@ -27,6 +27,21 @@ def coord_as_bin_edges(da, key, dim=None):
         return concat([left, center, right], dim)
 
 
+def repeat(x, dim, n):
+    """
+    Equivalent of Numpy's `repeat` function: repeat the values of a variable `n` times
+    along dimension `dim`.
+    """
+    index = x.dims.index(dim) + 1
+    dummy_dim = '_'
```

Comment:
This is going to break if that matches a dim name, use `uuid.uudi4()` (grep in `scipp` to see how we do this in other places).